src/PollingBlockTracker.test.ts

```diff
@@ -984,6 +984,180 @@ describe('PollingBlockTracker', () => {
         );
       });
     });
+
+    describe('with useCache: false', () => {
+      describe('when the block tracker is not running', () => {
+        it('should not fetch a new block even if a block is already cached and less than the polling interval time has passed since the last call', async () => {
+          recordCallsToSetTimeout();
+
+          await withPollingBlockTracker(
+            {
+              provider: {
+                stubs: [
+                  {
+                    methodName: 'eth_blockNumber',
+                    result: '0x1',
+                  },
+                  {
+                    methodName: 'eth_blockNumber',
+                    result: '0x2',
+                  },
+                ],
+              },
+            },
+            async ({ blockTracker }) => {
+              await blockTracker.getLatestBlock();
+              const block = await blockTracker.getLatestBlock({
+                useCache: false,
+              });
+              expect(block).toBe('0x1');
+              expect(blockTracker.isRunning()).toBe(false);
+            },
+          );
+        });
+
+        it('should fetch a new block even if a block is already cached and more than the polling interval time has passed since the last call', async () => {
+          recordCallsToSetTimeout({
+            numAutomaticCalls: 1,
+          });
+
+          await withPollingBlockTracker(
+            {
+              provider: {
+                stubs: [
+                  {
+                    methodName: 'eth_blockNumber',
+                    result: '0x1',
+                  },
+                  {
+                    methodName: 'eth_blockNumber',
+                    result: '0x2',
+                  },
+                ],
+              },
+            },
+            async ({ blockTracker }) => {
+              await blockTracker.getLatestBlock();
+              const block = await blockTracker.getLatestBlock({
+                useCache: false,
+              });
+              expect(block).toBe('0x2');
+              expect(blockTracker.isRunning()).toBe(false);
+            },
+          );
+        });
+      });
+
+      describe('when the block tracker is already started', () => {
```

Comment:
not confident i'm mixing numAutomaticCalls and interceptCallback correctly for the two tests in this scenario